Jenkinsfile

```diff
@@ -6,8 +6,7 @@ podTemplate(label: label,
                                            resourceRequestCpu: '1000m',
                                            resourceLimitCpu: '3800m',
                                            resourceLimitMemory: '3500Mi',
-                                           envVars: [secretEnvVar(key: 'TEST_API_KEY_WRITE', secretName: 'jetfire-test-api-key', secretKey: 'playgroundApiKey'),
-                                                     secretEnvVar(key: 'TEST_API_KEY_READ', secretName: 'jetfire-test-api-key', secretKey: 'playgroundApiKey'),
+                                           envVars: [secretEnvVar(key: 'TEST_API_KEY', secretName: 'jetfire-test-api-key', secretKey: 'playgroundApiKey'),
```

Comment:
https://github.com/cognitedata/cognite-sdk-scala/commit/61921cde2e00420bd777663605a77301855d12d7

I don't think this branch is up to date with master.

Reply:
Rebasing while merging can be done, right ?

Reply:
You need to rebase locally and do a force push (with lease) to the PR branch.